apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -88,7 +89,12 @@ const CreateEventPage = ({ history }: CreateEventProps) => {
           return diff <= 30 * DAY_IN_MILLISECONDS;
         }),
       startTime: Yup.string().required('Start Time is required.'),
-      endTime: Yup.string().required('End Time is required.'),
+      endTime: Yup.string()
+        .required('End Time is required.')
+        .test('is-greater', 'The end time must be after the start time.', function (endTime) {
+          const { startTime } = this.parent;
+          return moment(endTime, 'HH:mm').isSameOrAfter(moment(startTime, 'HH:mm'));
+        }),
```

Comment:
This is valid if the event is one day long. However, if the event takes place over multiple days then it might, for example, start at 5 pm on one day and finish at 10 am the next.

I'm not sure if its possible, but I would try to combine (startTime + startDate) => start and (endTime + endDate) => end and then comparing start and end.

Reply:
Or maybe call this function only when startDate == endDate?

Reply:
true, that's a lot simpler

Reply:
ah true, I'm going to fix this, nice catch 👍